ca/django_ca/utils.py

```diff
@@ -965,8 +967,10 @@ def parse_encoding(value: Optional[Union[str, Encoding]] = None) -> Encoding:
 def parse_expires(expires: Expires = None) -> datetime:
     """Parse a value specifying an expiry into a concrete datetime."""
 
-    now = datetime.utcnow().replace(second=0, microsecond=0)
+    now = datetime.now(timezone.utc).replace(second=0, microsecond=0)
 
+    if not expires.tzinfo:
+        expires = expires.replace(tzinfo=get_current_timezone())
```

Comment:
Can alternatively be changed to `timezone.utc`

Reply:
yes, UTC sounds better here.